src/core/namespace.c

```diff
@@ -469,6 +469,32 @@ static void drop_outside_root(const char *root_directory, MountEntry *m, unsigne
         *n = t - m;
 }
 
+static void mount_mkdir_apivfs(const char *root_directory, MountEntry *m)
+{
```

Comment:
nitpick: we usually place the opening bracket on the same line as the function name